docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+# This starts databases for local development or integration tests as if they were launched locally.
+# Run `docker-compose up -d`
+# Shut down and clean up: `docker-compose down`
+version: "3"
```

Comment:
In the next PR also add MariaDB as a 4th database here to run the test for it (even if they are the same as MySQL).

Also, is `alpine` the version for Postgres or just the linux-distro associated with the image? if the latter, it might be worth adding a specific version (say the last or second to last LTS release), but whatever, not high priority if it's difficult to find.

Reply:
It's a linux distro, but its better to add the version too